artifactory_cleanup/rules/base.py

```diff
@@ -200,7 +200,12 @@ def filter(self, artifacts):
         return artifacts
 
     def delete(self, artifact, destroy):
-        artifact_path = quote("{repo}/{path}/{name}".format(**artifact))
+        if artifact["path"] == ".":
+            artifact_path = "{repo}/{name}".format(**artifact)
+        else:
+            artifact_path = "{repo}/{path}/{name}".format(**artifact)
+
+        artifact_path = quote(artifact)
```

Comment:
should be `artifact_path = quote(artifact_path)`

Reply:
You're so right, thank you for noticing such a bad mistake!